modules/accredited_representative_portal/spec/requests/accredited_representative_portal/application_spec.rb

```diff
@@ -29,34 +29,45 @@
       Rails.application.reload_routes!
     end
 
-    context 'when authenticated' do
-      context 'with a valid audience' do
-        it 'allows access' do
-          expect(subject).to have_http_status(:ok)
-        end
-      end
+    context 'when feature flag is enabled' do
+      before { Flipper.enable(:accredited_representative_portal_pilot) }
```

Comment:
Can you please see the features toggles docs regarding the preferred method to use Flipper in specs -> https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample

      # Recommended to limit the feature flag state to the scope of this test
      allow(Flipper).to receive(:enabled?).with(:feature_flag).and_return(true)
      # Not recommended. Affects global state and can have unintended side effects when tests run in parallel
      Flipper.enable(:feature_flag)

Reply:
Ah right. Thanks for noticing this @rjohnson2011